main.js

```diff
@@ -176,24 +181,27 @@ const renderFooter = () => {
 };
 
 const pinRepoEvent = () => {
-  document.querySelector("#pinnedRepos").addEventListener("click", (e) => {
+  document.querySelector("#repoForm").addEventListener("click", (e) => {
     if (e.target.id === "repoModal-btn") {
       let repoList = "";
       for (const i of repo) {
         repoList += `<li class="list-group-item">
-        <input id="repo${i.id}" class="form-check-input me-1" type="checkbox" value="..." aria-label="...">
+        <input id="${i.id}" class="form-check-input me-1" name="repos" type="checkbox" value="${i.name}" aria-label="...">
         ${i.name}
       </li>`;
       }
       renderToDom("#repoList", repoList);
     }
-    if (e.target.id === "savePinned") {
-      console.log(repoList);
-      var numFour = document.querySelector("#repo4");
-      if (numFour.checked) {
-        console.log("Susan checked the box");
-      } else {
-        console.log("no check");
+    if (e.target.id === "savePinned-btn") {
+      const checkboxes = document.getElementsByName("repos");
+      for (let i = 0; i < checkboxes.length; i++) {
```

Comment:
Huh... Interesting way of going about doing this 🤔  I wonder if you have since learned another way to solve this problem yet.